packages/wallets/src/evm/connectors/device-wallet/index.ts

```diff
@@ -1,15 +1,16 @@
+import { Chain, normalizeChainId } from "../../../lib/wagmi-core";
 import { ConnectParams, TWConnector } from "../../interfaces/tw-connector";
 import type {
   DeviceWalletConnectionArgs,
   DeviceWalletImpl,
 } from "../../wallets/device-wallet";
-import type { Chain } from "@thirdweb-dev/chains";
 import type { Signer } from "ethers";
 import { providers } from "ethers";
 
 export type DeviceWalletConnectorOptions = {
-  chain: Pick<Chain, "chainId" | "rpc">;
+  chain: Chain;
   wallet: DeviceWalletImpl;
+  chains: Chain[];
```

Comment:
prob don't need this, should just need one chain at a time

if we make `switchChain()` take a chain object it should work nicely